src/entities/DealTokenSwap.ts

```diff
@@ -204,6 +221,42 @@ export class DealTokenSwap implements IDeal {
     return this.fundingWasInitiated && !this.isExecuted && !this.fundingPeriodHasExpired;
   }
 
+  /**
+   * Gets the DAO based on the connected wallet address
+   */
+  @computedFrom("ethereumService.defaultAccountAddress")
```

Comment:
"Wallet" isn't a term we use so much in the code.  Can we use "Account", just to be consistent?  I know that sometimes we (I) use "user", which is also inconsistent.  But at least we can avoid adding a third synonym?

Reply:
I changed all references from "Wallet" to "Account"